src/main/java/org/sunbird/workflow/service/impl/NotificationServiceImpl.java

```diff
@@ -169,11 +174,67 @@ public void sendEmailNotification(WfRequest wfRequest) {
 		}
 	}
 
+	public void sendNotificationToMdoAdmin(WfRequest wfRequest) {
+		WfStatusEntity wfStatusEntity = wfStatusRepo.findByApplicationIdAndWfId(wfRequest.getApplicationId(),
+				wfRequest.getWfId());
+		WfStatus wfStatus = workflowservice.getWorkflowStates(wfStatusEntity.getRootOrg(), wfStatusEntity.getOrg(),
+				wfStatusEntity.getServiceName(), wfStatusEntity.getCurrentStatus());
+		try {
+			logger.info("Notification workflow status entity, {}", mapper.writeValueAsString(wfStatusEntity));
+			logger.info("Notification workflow status model, {}", mapper.writeValueAsString(wfStatus));
+		} catch (JsonProcessingException e) {
+			e.printStackTrace();
+		}
+		if (!ObjectUtils.isEmpty(wfStatus.getNotificationEnable()) && wfStatus.getNotificationEnable()) {
```

Comment:
Need to check the status... make sure notification sent only when user created the request... 
in case of rejected / approved - notification must not be sent to admin

Reply:
if (!ObjectUtils.isEmpty(wfStatus.getNotificationEnable()) && wfStatus.getNotificationEnable()
				&& !Arrays.asList(Constants.REJECTED, Constants.APPROVED).contains(wfStatus.getState().toUpperCase())) {


Is it ok for this situation